.github/workflows/frontend-check.yaml

```diff
@@ -49,3 +53,10 @@ jobs:
         env:
           # pass GitHub token to detect new build vs re-run build
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+      - name: Save screenshot
```

Comment:
Nit -- but I believe your current Cypress script saves at least *two*, and we probably will want to add more, so "screenshots" might be better, or even "Save Cypress test screenshots".

Reply:
did it